### DISCUSSION TITLE
Slots Image Component

### DISCUSSION THREAD

**Discussion by JoseAntonioNavasSolutia** [primefaces/community#1372] [PrimeVue] [answered]
I want to add a custom button to download an image but the component does not have the possibility to add buttons to the button panel.

**Comment by tugcekucukoglu** [comment, marked as answer]
What kind of UI are you planning exactly? Maybe you could try to create your own Image component using PrimeVue Image.

**Comment by JoseAntonioNavasSolutia** [comment]
![Captura](https://github.com/primefaces/community/assets/86769899/50ef7719-6693-43b6-a91d-311853641b67)
I want a custom button for download image
